src/components/Dropdown/Menu/Item/Item.module.css

```diff
@@ -1,36 +1,15 @@
 .Item {
-  background: transparent none;
```

Comment:
I'm now defaulting `Dropdown.Menu.Item` to use `Button` components with `level="text"`, which eliminates the need for a bunch of this CSS.